htmresearch/frameworks/nlp/imbu.py

```diff
@@ -145,6 +147,11 @@ def _modelFactory(self, modelType, **kwargs):
                     prepData=False,
                     retinaScaling=1.0)
 
+    elif modelType == "Keywords":
+      # k should be the number of data samples (i.e. numLabels) because Keywords
```

Comment:
This comment doesn't make sense. Do you mean to say "k should be the number of categories"